sections/basics/coming-from-css.js

```diff
@@ -60,6 +60,34 @@ component \`Counter\` and the Styled Component \`StyledCounter\`
 don't clash names but remain easily identifiable in the React Developer 
 Tools and Web Inspector.
 
+### Define Styled Components outside of the render method
+
+It is important to define your styled components outside of render method, 
+otherwise the styled component will be recreated on every single render pass.
+Defining a styled component within the render method is bad for performance
```

Comment:
~~is bad for performance~~ will thwart caching and drastically slow down rendering speed,